.github/workflows/linux.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.15.x', '1.14.x', '1.13.x' ]
+        go: [ '1.16.x', '1.15.x', '1.14.x' ]
```

Comment:
if we want to move to 1.16 and above, can we fix this? Currently, this line is inconsistent with https://github.com/theparanoids/crypki/pull/65/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R28